src/ripple/app/tx/impl/apply.cpp

```diff
@@ -55,10 +55,82 @@ invoke_preflight (PreflightContext const& ctx)
     case ttAMENDMENT:
     case ttFEE:             return Change           ::preflight(ctx);
     default:
+        assert(false);
         return temUNKNOWN;
     }
 }
 
+/*
+    invoke_preclaim<Tx> uses name hiding to accomplish
+    compile-time polymorphism of (presumably) static
+    class functions for Transactor and derived classes.
+*/
+template<class Tx>
```

Comment:
I used `Tx` to indicate that it the template class was related to `Transactor`, but if it matters, I'll change it to `T`.